### ISSUE TITLE
Define API appid and appsecret in configuration

### ISSUE THREAD

**Issue by AliakseiMat** [emqx/emqx#2621]
#### Environment

- OS: 
- Erlang/OTP: 
- EMQ: 

#### Description

The docs say "You can create an application in Dashboard, using appid and appsecret to authenticate". 
Is there the way predefine appid and appsecret before starting the broker in some config file? The problem is that we have 2 clusters and if we create creds in Dashboard then they will be the different, but we need the same creds for both clusters. And also we don't want to create them manually. Thank you!

**Comment by Gilbert-Wong** [comment]
Sorry, @AliakseiMat, the feature you want is not general requirements. it is business logic. We won't fix it. And if you persist with this feature, please contact @zhengyupan for enterprise support.
